packages/iTwinUI-react/src/core/Header/HeaderDropdownButton.tsx

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { DropdownMenu } from '../DropdownMenu';
+import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
+import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
+
+import { PolymorphicForwardRefComponent, useTheme } from '../utils';
+import { DropdownButtonProps } from '../Buttons';
+import { useMergedRefs } from '../utils';
+export type HeaderDropdownButtonProps = DropdownButtonProps;
+type HeaderDropdownButtonComponent = PolymorphicForwardRefComponent<
+  'button',
+  HeaderDropdownButtonProps
+>;
+
+/**
+ * Dropdown button component for Header
+ *
```

Comment:
Remove this line. In other components too.

Reply:
Removed